docs/actions.md

```diff
@@ -1,182 +1,117 @@
 # Actions
 
-Use [actions](/docs/api.md#actions) to update the state.
+Actions are functions which take the current [state](/docs/state.md) and return a partial state or a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a partial state. Actions represent the intent to manipulate the state tree.
```

Comment:
The answer is "won't", but i believe it isn't the correct question. This description isn't very clear, because it don't mention that you may return a function which gets one argument called `update` which may/should be used to update the state, and also that `update` function returns the merged new object.